### PR TITLE
Revise descriptions, add tags, casync to cryptography.

### PR DESCRIPTION
**Pull request by mvincerx**
Signed-off-by: Michael Vincerra <michael.vincerra@intel.com>

### CODE REVIEW

File: bundles/cloud-control [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # [TITLE]: cloud-control
-# [DESCRIPTION]: Run a cloud orchestration server
+# [DESCRIPTION]: Run a cloud orchestration server.
 # [STATUS]: Active
 # [CAPABILITIES]:
-# [TAGS]:
+# [TAGS]: Data Science, Developer Tools,
```
Comment (bryteise) [inline bundles/cloud-control]: hanging comma, I'll fix it.

File: bundles/containers-basic [hunk 2]
```diff
@@ -1,8 +1,8 @@
 # [TITLE]: containers-basic
-# [DESCRIPTION]: Run container applications from Dockerhub
+# [DESCRIPTION]: Run container applications from Dockerhub.
 # [STATUS]: Active
 # [CAPABILITIES]:
-# [TAGS]:
+# [TAGS]: Developer Tools, Data Science
```
Comment (bryteise) [inline bundles/containers-basic]: Data Science doesn't apply here.

File: bundles/containers-basic-dev [hunk 3]
```diff
@@ -2,7 +2,7 @@
 # [DESCRIPTION]: All packages required to build the containers-basic bundle.
 # [STATUS]: Active
 # [CAPABILITIES]:
-# [TAGS]:
+# [TAGS]: Developer Tools, Data Science
```
Comment (bryteise) [inline bundles/containers-basic-dev]: data science doesn't apply.

File: bundles/containers-virt [hunk 4]
```diff
@@ -1,8 +1,8 @@
 # [TITLE]: containers-virt
-# [DESCRIPTION]: Run container applications from Dockerhub in lightweight virtual machines
+# [DESCRIPTION]: Run container applications from Dockerhub in lightweight virtual machines.
 # [STATUS]: Active
 # [CAPABILITIES]:
-# [TAGS]:
+# [TAGS]: Developer Tools, Data Science
```
Comment (bryteise) [inline bundles/containers-virt]: data science doesn't apply

File: bundles/containers-virt-dev [hunk 5]
```diff
@@ -2,7 +2,7 @@
 # [DESCRIPTION]: All packages required to build the containers-virt bundle.
 # [STATUS]: Active
 # [CAPABILITIES]:
-# [TAGS]:
+# [TAGS]: Developer Tools, Data Science
```
Comment (bryteise) [inline bundles/containers-virt-dev]: data science doesn't apply.

**Comment by bryteise** [comment]
This needs a little revision.

File: bundles/cloud-control (hunk 1 above)
Comment (bryteise) [reply bundles/cloud-control]: Since there are other things to work on in this PR, could you fix this too?

File: bundles/cloud-api [hunk 6]
```diff
@@ -1,8 +1,8 @@
 # [TITLE]: cloud-api
-# [DESCRIPTION]: API helpers for cloud access
+# [DESCRIPTION]: API helpers for cloud access. 
 # [STATUS]: Active
 # [CAPABILITIES]:
-# [TAGS]:
+# [TAGS]: Data Science, Developer Tools
```
Comment (bryteise) [inline bundles/cloud-api]: Data science shouldn't be here

File: bundles/cloud-native-basic [hunk 7]
```diff
@@ -1,8 +1,8 @@
 # [TITLE]: cloud-native-basic
-# [DESCRIPTION]: Contains ClearLinux native software for Cloud
+# [DESCRIPTION]: Contains Clear Linux* OS-native software for cloud.
 # [STATUS]: WIP
 # [CAPABILITIES]:
-# [TAGS]:
+# [TAGS]: Data Science, Developer Tools
```
Comment (bryteise) [inline bundles/cloud-native-basic]: Data science shouldn't be here

**Comment by bryteise** [comment]
Sorry I must have missed two of the data science mentions.

File: bundles/containers-basic (hunk 2 above)
Comment (bryteise) [reply bundles/containers-basic]: Still an issue.

File: bundles/containers-basic-dev (hunk 3 above)
Comment (bryteise) [reply bundles/containers-basic-dev]: Still an issue.

**Comment by ahkok** [comment]
b70e2b3d9f5e483ab0fb207d328b214c5f016e9c

**Pull request closed without merging**